package.json

```diff
@@ -43,9 +43,11 @@
   "author": "Yoast",
   "license": "GPL-3.0",
   "dependencies": {
+    "a11y-speak": "git+https://github.com/Yoast/a11y-speak.git#master",
     "@wordpress/a11y": "^1.0.7",
     "@wordpress/i18n": "^1.1.0",
     "algoliasearch": "^3.22.3",
+    "clipboard": "^1.5.15",
```

Comment:
This should not have any effect. The package on npm specifies `ClipboardJS` as the API. We never use that. We use `Clipboard` which is a browser API.

Reply:
We had some issues with running `yarn test` that revolved around clipboard though. We may have misinterpreted the error though. Can you advise on what we should do instead?